ecommerce_api_client/tests/test_auth.py

```diff
@@ -18,20 +25,32 @@ def setUp(self):
         self.signing_key = 'edx'
         httpretty.register_uri(httpretty.GET, self.url)
 
-    def assert_expected_token_value(self, tracking_context=None):
+    def assert_expected_token_value(self, tracking_context=None, issuer=None, expires_in=None):
         """ DRY helper. """
 
         # Mock the HTTP response and issue the request
-        auth_kwargs = {'tracking_context': tracking_context} if tracking_context else {}
-        requests.get(self.url, auth=JwtAuth(self.username, self.full_name, self.email, self.signing_key, **auth_kwargs))
+        auth_kwargs = {'expires_in': expires_in}
 
-        # Verify the header was set as expected on the request
         data = {
```

Comment:
nitpick: renaming this variable to something like `signing_data` would make for an easier read.